python/ray/tests/test_logging.py

```diff
@@ -190,7 +190,8 @@ def component_and_suffix_exists(component, paths):
 def test_log_rotation(shutdown_only, monkeypatch):
     max_bytes = 1
     backup_count = 3
-    set_logging_config(monkeypatch, max_bytes, backup_count)
+    os.environ["RAY_ROTATION_MAX_BYTES"] = str(max_bytes)
```

Comment:
Use monkeypatch to update environ, so that environ can be auto reverted to old value when the test finishes

Reply:
Actually why not calling `set_logging_config`?

Reply:
It's been a while, not sure why at that time... reverted back.